### PR TITLE
Modified verification to use fingerprints

### PR DESCRIPTION
**Pull request by sabre1041**
Modified one of the verification parameters to use fingerprints instead of comparing content

### CODE REVIEW

File: pkg/pgp/pgp.go [hunk 1]
```diff
@@ -72,6 +73,18 @@ func ExtractPublicKey(entity *openpgp.Entity) ([]byte, error) {
 
 }
 
+func GetFingerprintFromPublicKey(content []byte) (string, error) {
+
```
Comment (cpanato) [inline pkg/pgp/pgp.go:77]: nit: remove this extra line

File: pkg/verifier/verify.go [hunk 2]
```diff
@@ -55,10 +56,22 @@ func (v *Verifier) VerifyRekor() error {
...
-		return errors.New("Error Comparing Public Key: Value Stored in Rekor Does Not Match Key Used to Sign Provenance File")
+	// Verify Public Key Fingerprints and Signature
+	rekorFingerprint, err := pgp.GetFingerprintFromPublicKey([]byte(v.Entry.HelmObj.PublicKey.Content))
+
+	if err != nil {
+		return errors.New("Failed to obtain fingerprint from Rekor public key")
+	}
+
+	providedPublicKey, err := pgp.GetFingerprintFromPublicKey(v.PublicKey)
+
```
Comment (hectorj2f) [inline pkg/verifier/verify.go:67]: nit: remove extra line.

Comment (sabre1041) [reply pkg/verifier/verify.go]: Fixed

File: pkg/pgp/pgp.go (hunk 1 above)
Comment (sabre1041) [reply pkg/pgp/pgp.go:77]: The inclusion of a space at the beginning/end of method matches the rest of the application. We can remove the spaces across the board in a future enhancement

**Review by hectorj2f — APPROVED** [review]

**Review by cpanato — APPROVED** [review]

**Pull request merged**